modules/dxp/apps/commerce/commerce-organization-web/src/main/resources/META-INF/resources/js/ManagementBar/ResultsBar.tsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
```

Comment:
Is it correct the copyright? In other new files I see 2024

Reply:
Weird that it didn't autofix the year.